src/main/kotlin/com/sipios/springsearch/strategies/ParsingStrategy.kt

```diff
@@ -24,13 +24,16 @@ interface ParsingStrategy {
             SearchOperation.NOT_EQUALS -> builder.notEqual(path.get<Any>(fieldName), value)
             SearchOperation.STARTS_WITH -> builder.like(path.get(fieldName), "$value%")
             SearchOperation.ENDS_WITH -> builder.like(path.get(fieldName), "%$value")
-            SearchOperation.CONTAINS -> builder.like((path.get<String>(fieldName).`as`(String::class.java)), "%$value%")
+            SearchOperation.CONTAINS -> {
```

Comment:
Not a big fan of this as it will be a breaking change. Would rather add a ICONTAINS operator, what do you think?

Reply:
Also we need to adapt the tests for this

Reply:
I don’t think it will be a breaking change as the branch im merging with is not yet public; but we could have a different searchoperator which distinguishes between exact matches and ignorecase.

Reply:
Also I think the tests work fine for it right now, they don't give any errors.